File: Easy/JavaScript/2490. Circular Sentence.js
var isCircularSentence = function(sentence) {
    for(let i = 0; i < sentence.length; i++){
        if(sentence[i] === ' '){
            if(sentence[i - 1] !== sentence[i + 1])
                return false;
        }
    }

    return sentence[0] === sentence[sentence.length - 1];
};

File: Easy/Python/2490. Circular Sentence.py
class Solution(object):
    def isCircularSentence(self, sentence):
        for i in range(len(sentence)):
            if sentence[i] == ' ':
                if sentence[i - 1] != sentence[i + 1]:
                    return False
        
        return sentence[0] == sentence[len(sentence) - 1]
